### PR TITLE
CSS updates into main

### PR DESCRIPTION
**Pull request by ceramic-cat**
Tropic
image

and nordic!
image

### CODE REVIEW

File: design.css [hunk 1]
```diff
@@ -1,24 +1,54 @@
...
+}
+
+
+
 header {
   display: flex;
   justify-content: end;
 }
 
+.description{
```
Comment (Pandangeloo) [inline design.css:42]: Why is it two of the same? .description?

File: index.html [hunk 2]
```diff
@@ -30,48 +30,50 @@
...
+                    <option class="dropdown-value" value="borås">Borås</option>
+                    <option class="dropdown-value" value="norrkoping">Norrköping</option>
+                    <option class="dropdown-value" value="dodoma">Dodoma</option>
+                    <option class="dropdown-value" value="zanzibar">Zanzibar</option>
                 </select>
                 
                 <!--Visar väderinfo (fylls med data från js) -->
                 <div id="weatherContainerA" class="weather-container">
-                  <h2 data-weather="city">Stad</h2> <!-- Stad -->
+                  <h2 data-weather="city" class="cityName">Stad</h2> <!-- Stad -->
```
Comment (Pandangeloo) [inline index.html:48]: @emmamjs-7  didnt we change stad to a #?

**Comment by Pandangeloo** [comment]
We also need to decide if we should write in english or swedish on page. Because now it is very much a mix.

File: index.html [hunk 3]
```diff
@@ -30,48 +30,50 @@
...
                 <select class="btn bg-success-subtle dropdown-toggle"  data-bs-toggle="dropdown"
                     aria-expanded="false" name="chooseCityA" id="city-dropdown-a" 
                     onchange="fetchWeatherData('city-dropdown-a', 'weatherContainerA')">
-                    <option value="" disabled selected>Choose a city</option>
-                    <option value="boras">Borås</option>
-                    <option value="norrkoping">Norrköping</option>
-                    <option value="dodoma">Dodoma</option>
-                    <option value="zanzibar">Zanzibar</option>
+                    <option class="dropdown-value disabled" value="" disabled selected>Choose a city</option>
+                    <option class="dropdown-value" value="borås">Borås</option>
```
[the remark this answers is not in the document]
Comment (ceramic-cat) [reply index.html:40]: Bah, i fix

[the remark this answers is not in the document]
Comment (ceramic-cat) [reply index.html]: Fixed in new commit

File: design.css (hunk 1 above)
Comment (ceramic-cat) [reply design.css:42]: Most likely a mistake in the merge... Fixed in new commit

File: index.html (hunk 2 above)
Comment (Pandangeloo) [reply index.html:48]: I confused it with another thing. So this comment should be ignored. :)

File: index.html [hunk 4]
```diff
@@ -30,48 +30,50 @@
...
                     class="btn bg-success-subtle dropdown-toggle" data-bs-toggle="dropdown"
                     aria-expanded="false" name="chooseCityB" id="city-dropdown-b"
                     onchange="fetchWeatherData('city-dropdown-b', 'weatherContainerB')">
-                    <option value="" disabled selected>Choose a city </option>
-                    <option value="boras">Borås </option>
-                    <option value="norrkoping">Norrköping </option>
-                    <option value="dodoma">Dodoma </option>
-                    <option value="zanzibar">Zanzibar </option>
+                    <option class="dropdown-value disabled" value="" disabled selected>Choose a city</option>
+                    <option class="dropdown-value" value="borås">Borås</option>
```
Comment (Pandangeloo) [inline index.html:65]: Oh no! another borås!

**Review by Pandangeloo — APPROVED** [review]
Everything looks good!

File: index.html (hunk 4 above)
Comment (Pandangeloo) [reply index.html]: Yes! Stand down soldier! :D 👯‍♀️

**Pull request merged**